hw_i2c/sample-implementations/arduino/README.md

```diff
@@ -0,0 +1,117 @@
+# Arduino Specific README
+This guide contains some hints on getting the generic reference drivers working
+on Arduino platforms. Note that this is a generic driver meant to be adaptable
+to any platform with minimal effort but it is not, by design, a ready-to-use
+Arduino driver.
+
+Kindly keep in mind that Sensirion cannot assist with general Arduino questions
+not related to the driver.
+
+## Project Setup
+Arduino compiles all files in a Library (See
+[Build-Process](https://github.com/arduino/Arduino/wiki/Build-Process)). Thus,
+for Arduino to pick up the driver as a library all files must be in a directory
+with the name of that library and it must be in the folder where your Arduino
+IDE installs libraries.
+
+For the sake of this tutorial we'll call this the `sxx30` folder, where _sxx_ is
+substituted for the actual project name (name of your _.ino_ file). We suggest
+using the driver's name as project name e.g. _sgp30_, _sps30_, _shtc1_ etc.
+
+### Files to copy to your Library folder
+Use as base the *release* version of your driver from `make release` or, if
+you're not sure what this means, download the release Zip from the driver's
+releases page on GitHub, e.g. for the embedded-sgp driver download
+_sgp30-x.y.z.zip_ from
+[https://github.com/Sensirion/embedded-sgp/releases](https://github.com/Sensirion/embedded-sgp/releases)
+
+* Copy all content except the example usage file (`example_usage.c` or
+  `sxx30_example_usage.c`) from the release folder into the `sxx30` library
+  folder.
+* Copy the Arduino sample implementation
+  `sensirion_hw_i2c_implementation.cpp` from this folder into the `sxx30`
```

Comment:
Should we add instructions on how to use the sw_i2c implementation? Since some Arduino boards have a limited buffer. See https://github.com/Sensirion/embedded-sps/issues/14 and https://github.com/Sensirion/embedded-sps/issues/7